Builds/Test.py

```diff
@@ -61,6 +66,41 @@ def powerset(iterable):
 IS_WINDOWS = platform.system().lower() == 'windows'
 IS_OS_X = platform.system().lower() == 'darwin'
 
+# CMake
+
+CMAKE_UNITY_TARGETS = ['rippled', 'rippled_classic']
+CMAKE_NONUNITY_TARGETS = ['rippled', 'rippled_unity']
```

Comment:
This has to do with a decision we made for the cmake config specifically to support both unity and non-unity builds in a single solution in the Visual Studio IDE. If you target a unity build, `rippled` will be a unity build, and as a convenience, `rippled_classic` will be available as a nonunity build in the solution. Conversely, if you target a non-unity build, `rippled` will be a non-unity build, and `rippled_unity` will be the convenience.